android/src/main/java/cl/json/ShareFiles.java

```diff
@@ -79,7 +77,7 @@ private boolean isBase64File(Uri uri) {
             String type = uri.getSchemeSpecificPart().substring(0, uri.getSchemeSpecificPart().indexOf(";"));
             if (this.intentType == null) {
                 this.intentType = type;
-            } else if (type != null && !this.intentType.equalsIgnoreCase(type) && this.intentType.split("/")[0].equalsIgnoreCase((type.split("/"))[0])) {
```

Comment:
Code path analysis indicates it will never be null, it was a no-op. I suppose the substring method will either return something, empty string, or throw.